docs/style-guide/utilities.md

```diff
@@ -171,6 +171,33 @@ For each size in our [type definitions](/style-guide/type.html) we include a uti
 
 We support both single line truncation as well as multi-line. Multi-line truncation is customizable with the following properties:
 
+- `MaxLineLimit` - the maximum number of allowed lines (defaults to `3`)
```

Comment:
question: Should this CSS variable be named something to indicate it's related to truncation? 

Also, in the docs page, I don't see a mention that this is a CSS variable except in the example?